aspects/models/enrollment/enrollments_by_day.sql

```diff
@@ -5,6 +5,9 @@
 -- to Unix timestamps, then generating a range of Unix timestamps,
 -- creating one record per timestamp (via arrayJoin), and then converting
 -- the timestamps back into dates
+{{ config(
+  post_hook = ["{{apply_admin_rbac(this)}}", "{{apply_instructor_rbac(this)}}"]
+) }}
```

Comment:
This is what we would need to apply to all of the models that have course ids. It's possible we could do this on whole directories of models, which I think would probably be preferable. I haven't had time to look.